src/bus/policy.c

```diff
@@ -688,11 +695,13 @@ int policy_registry_import(PolicyRegistry *registry, CDVar *v) {
                         return error_fold(r);
         }
 
-        c_dvar_read(v, "]b)>", &apparmor);
+        c_dvar_read(v, "]bs)>", &apparmor, &bustype);
```

Comment:
I also fixed up `test/dbus/util-broker.c` so the test-suite should now correctly set the bustype as well.